Add new alignment tags to `BBCode` (#14645)
ppy/osu-wiki: wiki/BBCode/en.md

- `[centre][b]text[/b][/centre]` is correct
- `[b][centre]text[/b][/centre]` is incorrect

## Alignment

> Alignment tags are used to control the horizontal positioning of text and BBCode elements. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be aligned accordingly with the alignment tag, but the stylistic lines and such will not.

Reviewer: probably should be described more clearly

```suggestion
Alignment tags are used to control the horizontal positioning of text and BBCode elements. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be aligned accordingly with the alignment tag, but the vertical line of the quote block will remain on the left.
```

The `[centre]` tag is used to align text to the centre of a box. This is most often used for stylistic effect in titles, headers, or poems. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be centred, but the stylistic lines and such will not.

### Right

```
[right]text[/right]
```

> The `[right]` tag is used to align text to the right of a box. This is most often used for right-to-left languages alongside stylistic effect in titles, headers, or poems. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be right aligned, but the stylistic lines and such will not.

Reviewer: ```suggestion
The `[right]` tag is used to align text to the right of a box. This is most often used for right-to-left languages alongside stylistic effect in titles, headers, or poems. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be right-aligned, but the stylistic lines and such will not.
```

> The `[right]` tag is used to align text to the right of a box. This is most often used for right-to-left languages alongside stylistic effect in titles, headers, or poems. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be right aligned, but the stylistic lines and such will not.

Reviewer: what are "stylistic lines"

Reviewer 2: this means the vertical line at the left side of a quote block. This sentence is copied from the `[centre]`'s description right above.
